encoder.go

```diff
@@ -10,6 +10,9 @@ import (
 	"time"
 )
 
+// Base64 represents value in base64 encoding
+type Base64 string
```

Comment:
I don't see there this type is used? Do we need it?

Reply:
@kolo encoder.go:70